src/App.tsx

```diff
@@ -1,16 +1,23 @@
+import { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import { NewMovie } from './components/NewMovie';
 import moviesFromServer from './api/movies.json';
+import { Movie } from './types/Movie';
 
 export const App = () => {
+  const [movies, setMovies] = useState(moviesFromServer);
```

Comment:
```suggestion
  const [movies, setMovies] = useState<Movie>(moviesFromServer);
```
I guess it has any type now